forc/src/utils/defaults.rs

```diff
@@ -2,19 +2,21 @@
 /// the chance to insert some helpful comments and nicer formatting.
 pub(crate) fn default_manifest(project_name: &str) -> String {
     let real_name = whoami::realname();
+    let orga_name = "organizationName";
 
     format!(
         r#"[project]
 name = "{}"
 author = "{}"
+organization = "{}"
```

Comment:
Actually I think this should be removed from the default manifest file generated with `init.` It's not even being filled in with a reasonable value. The default manifest file doesn't _have_ to include every single possible field.

Reply:
Done